lib/workspace.js

```diff
@@ -336,13 +302,10 @@ module.exports = {
       questionId: question.id,
     });
 
-    // Update home directory location. We do this now to catch both first
-    // time startups and when the workspace is being reset.
-    const homedir_location = config.workspaceHomeDirLocation;
-    workspace.homedir_location = homedir_location;
+    // TODO: remove this code once nothing is reading from the `homedir_location` columns.
```

Comment:
This is necessary to get us through a deployment of this code; we'll need to continue writing this column for as long as there are hosts that could read it.